src/NServiceBus.Persistence.AzureStorage/NServiceBus.Persistence.AzureStorage.csproj

```diff
@@ -10,8 +10,8 @@
   <ItemGroup>
     <PackageReference Include="WindowsAzure.Storage" Version="[9.0.0, 10.0.0)" />
     <PackageReference Include="NServiceBus" Version="[7.0.0, 8.0.0)" />
-    <PackageReference Include="Fody" Version="3.1.4" PrivateAssets="All" />
-    <PackageReference Include="Obsolete.Fody" Version="4.4.1" PrivateAssets="All" />
+    <PackageReference Include="Fody" Version="6.1.0" PrivateAssets="All" />
```

Comment:
out of curiosity can this be made a transitive dependency instead? Since obsolete depends on Fody? https://www.nuget.org/packages/Obsolete.Fody/

cc  @bording

Reply:
We could make it a transitive dependency, but given how strict Simon is in following SemVer, we'd end up using a rather old/outdated version of Fody pretty quickly if we did that.